src/authorizations/users.rs

```diff
@@ -1,29 +1,33 @@
-fn authorized_users() -> Vec<String> {
-    dotenv::var("AUTHORIZED_USERS")
-        .unwrap()
-        .split(',')
-        .map(String::from)
-        .collect()
+use crate::config::config::Config;
+
+fn authorized_users(config: &Config) -> Vec<String> {
+    let split_string = config.authorized_users.split(',');
+    let auth_users: Vec<String> = split_string.map(String::from).collect();
```

Comment:
Instead of storing the list of users as a string in the config and parsing it every time, it would be better to parse it beforehand and store it in a `HashSet` in the config.

Reply:
Excellent point - done!